test_collections/matter/sdk_tests/support/performance_tests/scripts/sdk/TC_COMMISSIONING_1_0.py

```diff
@@ -57,10 +66,10 @@ def desc_TC_COMMISSIONING_1_0(self) -> str:
     def steps_TC_COMMISSIONING_1_0(self) -> list[TestStep]:
         steps = [TestStep(1, "Loop Commissioning ... 1")]
 
-        if len(self.additional_steps) > 0:
-            return self.additional_steps
-        else:
-            return steps
+        for i in range(2, self.iterations + 1):
+            steps.insert(i, TestStep(i, f"Loop Commissioning ... {i}"))
```

Comment:
@rquidute I had a change of heart on this part. Would be better do the below?
```suggestion
        steps = [TestStep(i, "Loop Commissioning ... {i}") for i in range(1, self.iterations + 2)]
```